pkg/must-gather/omc-client/utils.go

```diff
@@ -0,0 +1,105 @@
+package omcclient
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	git "github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing"
+)
+
+var omcGitUrl = "https://github.com/gmeghnag/omc.git"
+var omcGitTag = "v3.12.2"
+
+// getOmcCommandPath gets the path to the omc command. If the command is not found in the PATH,
+// it will be downloaded to the _bin directory in the git repository root and then return the
+// path to the command.
+func getOmcCommandPath() (string, error) {
+	// Check if omc is in the PATH
+	path, err := exec.LookPath("omc")
+	if err == nil {
+		return path, nil
+	}
+
+	// The omc command is not in the PATH, so we need to build it from source.
+	commandPath, err := buildFromSource(omcGitUrl, omcGitTag)
```

Comment:
should setting up omc client responsibility of mcp tool ?  I think it must reside outside of it.

Reply:
I am fine with it. One thing to keep in mind is that since ovsdb-tool uses omc to map the ovn dbs to the nodes, if omc is not available, then ovsdb-tool will also not work.

How should the MCP server handle this situation? Should it not fail initializing the server, or should we skip adding any of the mg tools?